indexer/kayak.cpp

```diff
@@ -0,0 +1,140 @@
+#include "indexer/kayak.hpp"
+
+#include "base/string_utils.hpp"
+
+#include "coding/url.hpp"
+
+#include <ctime>
+#include <map>
+#include <sstream>
+#include <string>
+
+#include "private.h"
+
```

Comment:
Put ```namespace osm``` here before usings and an anonimous namespace.

Reply:
I am not following. Swap "namespace {}" and "namespace osm {}" sections?

Reply:
ah, probably put "namespace {}" into "namespace osm {}"... ok